tests/scripts/depends.py

```diff
@@ -521,7 +523,7 @@ def main():
                             default='.')
         parser.add_argument('-k', '--keep-going',
                             help='Try all configurations even if some fail (default)',
-                            action='store_true', dest='keep_going', default=True)
+                            action='store_false', dest='keep_going')
```

Comment:
Now the documentation says default on, but the code says default off.

I think it would be nicer to keep the default on. When we run the CI, we want feedback on all configurations. The fact that the perl depends script stop on the first error is an annoyance, not a desired feature. I would prefer to just keep the code as is here.

Reply:
That's just a fix for a pre-existing misinterpretation of `store_true`: it stores passed `True` value, but has a default value of `False`.
`store_false` has a default value of `True`, and stores `False` if passed.
https://docs.python.org/3/library/argparse.html#action

Reply:
Ah, you're right, sorry. I'll revert that commit.

Reply:
Mutually exclusive would mean you can't have a wrapper script that adds one of them, and then call the wrapper script with an extra option to override it. Mutually exclusive options are usually a bad command line design: generally, subsequent options should override earlier ones.